{{Hamilton_McMaster}}

<html>	

<div class = "band">
<h2> Abstract </h2>
<div class = "highlightBox">
<p>Light-based bacterial protein expression systems have been well documented. Tabor <i>et al.</i> demonstrated a multichromatic protein expression system in E. coli through the combination of the CcaR-CcaS green-light sensitive construct derived from cyanobacteria and a previously characterized red-light sensitive system. This could be leveraged for the expression of multiple genes by exposing the bacteria to different wavelengths of light, i.e. red and green. We propose the application of this construct to recombinant protein expression. Using genetic cloning and recombinant protein techniques, <i>E. coli</i> is concurrently transformed with three plasmids: a chromophore, a red-light sensitive system, and a green-light sensitive system. Protein expression is induced through shining red light on the bacterial population. Following this, cell lysis is triggered by shining green light, which is mediated by a T4 Holin/Endolysin system. This releases the protein of interest into the cell media, allowing for straightforward collection and purification. This system would have important applications in both research and industry, allowing for the optimization and potential automation of heterologous bacterial protein production.</p>
</div>
<h2> Wet Lab </h2>
<div class = "highlightBox">
<p>The increasing prevalence of multidrug-resistant and hypervirulent bacterial strains represents a growing global healthcare concern. However, early detection of pathogenic microbes allow for timely care of patients and the prevention of infectious strains proliferation. In the face of the current challenges in profiling bacterial infections, we are designing a fast and user-friendly detection assay using fluorogenic DNAzymes as the molecular probe. Our fluorogenic DNAzymes are single-stranded functionalized DNA capable of cleaving a fluorophore-quencher construct specifically in the presence of E. coli. Upon cleavage, the quencher can no longer suppress the fluorophore, resulting in intense fluorescence. This fluorescence intensity can be also used to quantify the amount of E. coli, and potentially achieve strain-specific recognition. Our novel approach to early pathogen detection technology can potentially enhance our ability to respond to disease outbreaks from infectious bacteria.</p>
</div>
<h2> Human Practices </h2>
<div class = "highlightBox">
<p>The Human Practices team will investigate the current process of detecting antibiotic resistant bacteria to better understand the context for which the Wet Lab is developing the product. We will examine direct and indirect costs, accessibility, time constraints, safety and ethics of the current processes. From the information collected, we will discuss the need and the potential impact of developing products that will detect bacteria more cheaply and quickly. We will focus in Hamilton and surrounding areas, and conduct personal interviews with the stakeholders. Individuals that we are looking to contact are researchers, Hamilton residents, sanitation advocacy groups, politicians, administrators, and journalists. HP may build case studies that focus on patient groups such as seniors, infants and individuals with low immune response. HP will remain in heavy communication with the wet lab and dry lab teams, to ensure that the final products from the research teams are integrated to represent thorough exploration of antibiotic resistant bacteria detection.</p>
</div>
<h2> Dry Lab </h2>
<div class = "highlightBox">
<p>The Dry Lab will be supporting the Wet Lab as they use SELEX to find a DNAzyme for a highly pathogenic strain of C. Difficile. During the SELEX process, the Dry Lab will analyze deep sequencing data, monitoring sequence diversity as selection progresses to find possible homologies or motifs that are enriched through selection, using a clustering algorithm. After SELEX has narrowed the DNA library to a few potential DNAzymes, the Dry Lab will use a genetic algorithm as an in silico approach to find highly specific DNAzyme sequences. The affinities of the selected sequences will be measured by the Wet Lab, and high-affinity sequences will be preferentially crossed-over and mutated to create the next generation, in an iterative process that will produce higher-affinity DNAzymes.</p>
</div>
<h2> Community Outreach </h2>
<div class = "highlightBox">
<p>The iGEM community outreach team is focused on spreading the interesting work being done in the world of synthetic biology, and more specifically by our research team, across the Hamilton community. We put an extra emphasis on trying to educate and empower students who may be interested in exploring our area of research as a means of achieving two central goals. First, we hope to inform the members of our community about the possible applications of synthetic biology on a larger scale in order to garner more support and interest for the field. Secondly, and perhaps more importantly, we also are striving to give back to the very community that supports our work by providing unique opportunities to a variety of students. Our attempts to achieve these goals have manifested themselves through hosting a series of programs, from workshops to speaker series, that give us the best opportunity to spread the work of iGEM McMaster.</p>
</div>
</div>
</html>
{{:Template:McMasterU_Footer}}